content/stable-rc.html

```diff
@@ -7,6 +7,14 @@
 - general
 :author: kohsuke
 ---
+<html>
+<head>
+<meta charset="UTF-8">
+<meta name="description" content="Free Web tutorials">
```

Comment:
That was just for the owner to get to know about those things that they should be here and so they can fill up in their own more accurately because they know their site better than any other like keywords, description etc.

Reply:
The jenkins.io site contains over 1500 pages.  We don't persuade people to add meta information by adding incorrect meta information to a single page and hope that they will understand the value of creating and maintaining that meta information.  

If the meta information will significantly benefit real users of the jenkins.io site, then we would need to automate the creation of that meta information during site generation.

If you'd like to persuade contributors to the jenkins.io site of the value of meta tags, please start a discussion in the Jenkins documentation mailing list or in the Jenkins docs channel on Gitter.  A pull request with inaccurate and incomplete information is not an effective way to persuade authors to change their approach.